src/containers/LargeScaleStructureContainer.jsx

```diff
@@ -1,29 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import debounce from 'lodash/debounce';
-import Slider from 'react-md/lib/Sliders/Slider';
-import SelectionControl from 'react-md/lib/SelectionControls/SelectionControl';
+import classnames from 'classnames';
 import API from '../lib/API.js';
-import { extentFromSet, formatValue } from '../lib/utilities.js';
-import { arrayifyData } from '../components/charts/largeScaleStructure/largeScaleStructureUtilities.js';
-import LargeScaleStructure3D from '../components/charts/largeScaleStructure/LargeScaleStructure3D.jsx';
+import { colorize } from '../lib/utilities.js';
+import NavDrawer from '../components/charts/shared/navDrawer/index.jsx';
 import LargeScaleStructure2D from '../components/charts/largeScaleStructure/LargeScaleStructure2D.jsx';
+import LargeScaleStructure3D from '../components/charts/largeScaleStructure/LargeScaleStructure3D.jsx';
 import '../components/charts/largeScaleStructure/large-scale-structure-plot.module.scss';
 
+import {
+  drawerContainer,
+  navItemText,
+  scrambleItem,
+  linkActive,
+} from '../components/charts/hubblePlot/hubble-plot.module.scss';
+import { paddedDrawerInner } from '../components/charts/shared/navDrawer/nav-drawer.module.scss';
```

Comment:
good reuse of existing styles, however, in this case, because the whole NavDrawer is wider than the default this isn't padding the left hand side of the inner drawer as it should.  It's not too apparent when it's got lots of room to spread out across the full width of the screen but if you put it in our TwoCol layout or squish the window down a bit you'll see the y axis gets clipped off pretty quick.  Additionally, I think given the dark aesthetic of this visualization that, that it makes sense for the visualization to go all the way to the bottom and right hand edge of the card it's living inside (rather than have these lil white stripes along those sides).   Which is just part of the iterative process.  Just life when we never have any design ahead of  building the things.  No biggie.